WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -33,6 +33,7 @@
     public static final String AZTEC_EDITOR_NAME = "aztec";
     public static final String WP_STORIES_CREATOR_NAME = "wp_stories_creator";
     public static final String WP_STORIES_JETPACK_VERSION = "9.1";
+    public static final String WP_CONTACT_INFO_JETPACK_VERSION = "8.5";
```

Comment:
Hi @illusaen or @mzorz I have a question about the Jetpack version. I am looking at the release notes above and it specifies that the contact info block works on WP.com site and sites with JetPack 9.1 and above. But this value is 8.5 so I am just checking if this is expected i.e  I am checking to see if these versions should be the same or not. Thanks 🙇

Reply:
Good observation @jd-alexander ! My [findings](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3001#issuecomment-762303994) tell me the first version of Jetpack shipping with that block was 8.5, so the release notes should be updated accordingly 👍 @illusaen do you want to upadte it?

Reply:
Also another question @mzorz should the block be tested with vanilla debug or vanilla release ?

Reply:
I think both should work as long as it's vanilla - I tested vanillaDebug only but it should be the same (the productFlavors work this way)